docs/components/authentication/sign-up.mdx

```diff
@@ -101,10 +101,10 @@ All props below are optional.
 | Name | Type | Description |
 | --- | --- | --- |
 | `appearance` | `object` | Controls the overall look and feel of the component. See [Appearance](/docs/components/customization/overview) for more information. |
-| `routing` | `string` | The routing strategy for your pages.<br />Supported values are: <ul><li>hash (default): Hash-based routing.</li> <li>path: Path-based routing.</li> <li>virtual: Virtual-based routing</li></ul> |
+| `routing` | `string` | The routing strategy for your pages.<br />Supported values are: <ul><li>hash (default): Hash-based routing.</li> <li>path: Path-based routing.</li> <li>virtual: Virtual-based routing</li></ul>Note: If you are using Environment Variables for Next or Remix to specify your routes, this will override to 'path'.|
```

Comment:
```suggestion
| `routing` | `string` | The routing strategy for your pages.<br />Supported values are: <ul><li>hash (default): Hash-based routing.</li> <li>path: Path-based routing.</li> <li>virtual: Virtual-based routing</li></ul>Note: If you are using Environment Variables for Next.js or Remix to specify your routes, this will be set to `path`.|
```